package.json

```diff
@@ -28,7 +28,7 @@
   },
   "scripts": {
     "prepublish": "./resources/prepublish.sh",
-    "test": "npm run lint && npm run check && npm run testonly",
+    "test": "yarn lint && yarn check && yarn testonly",
```

Comment:
This is bad practice; non yarn users/environments will not be able to run the script. Yarn knows to substitute `npm run` for `yarn` automatically when running npm scripts.